testing/ncids-css-testing/stories/components/nci-header/nci-extended-desktop-search.stories.jsx

```diff
@@ -11,13 +11,18 @@ const html = `
 		<div class="nci-header__navbar">
 			${nciImgLogo}
 			<div class="nci-header-nav__secondary">
+				<button class="usa-button nci-header-mobilenav__open-btn">Menu</button>
 				<form action="#" aria-label="Sitewide" class="nci-header-search" method="get" role="search">
 					<label class="usa-sr-only" for="nci-header-search__field">
 						Search
 					</label>
 					<input class="usa-input" id="nci-header-search__field" type="search" name="search" style="border-right:1px solid #1b1b1b;">
-					<div class="nci-header-search__overlay"></div>
```

Comment:
can you confirm that all stories, the documentation site, and the example site are all using the same and correct HTML for search?